progress.go

```diff
@@ -86,34 +85,41 @@ func (p *Progress) AddBar(total int) *Bar {
 // Listen listens for updates and renders the progress bars
 func (p *Progress) Listen() {
 	p.lw.Out = p.Out
-	for {
-		select {
-		case <-p.stopChan:
-			return
-		default:
-			time.Sleep(p.RefreshInterval)
+
+	go func() {
+		for _ = range p.ticker.C {
```

Comment:
[ticker.Stop](https://golang.org/pkg/time/#Ticker.Stop) doesn't close the tick chan, which means this goroutine will never exit and cause a leak. This is easy to check by adding a log/panic statement between lines 65/96.